mem0/memory/telemetry.py

```diff
@@ -58,14 +69,10 @@ def capture_event(event_name, memory_instance, additional_data=None):
     if additional_data:
         event_data.update(additional_data)
 
-    telemetry.capture_event(event_name, event_data)
```

Comment:
Wait, I am not sure why you are removing this line?

Reply:
Drink and code accident, added it again